src/app/about/page.tsx

```diff
@@ -0,0 +1,140 @@
+import styles from './page.module.css';
+export default function AboutPage() {
+  return (
+    <article className={styles.root}>
+      <h1>Про проєкт "Корені"</h1>
+      <section>
+        <h2>Хто автор Коренів?</h2>
+        <p>
+          Привіт. Я – Віталій Перегончук, розробник ПЗ,{' '}
```

Comment:
```suggestion
          Привіт. Я – Віталій Перегончук, розробник ПЗ,&nbsp;
```

highly recommend to use special HTML characters for these cases

Reply:
@alexandrtovmach why would you have non-breaking space in that place?

Reply:
Yeah, I like it from DX perspective, but not sure the character is intended for this...

Reply:
that's right symbol for that case, trust FE dev)

Reply:
I added `&nbsp;` and `&#20;` where appropriate.